src/common/core/core.js

```diff
@@ -1100,6 +1148,16 @@ define([
         this.getValidAspectNames = core.getValidAspectNames;
 
         /**
+         * Returns the list of the META defined aspect names of the node that were specifically defined for the node.
+         * @param {module:Core~Node} node - the node in question.
+         *
+         * @return {string[]} The function returns the aspect names that were defined for the node.
```

Comment:
Language is not consistent with getOwnValidAttributeNames
`The function returns the aspect names that are specifically defined for the node.`